programmers/난이도별/level02.H-Index/dkdlelk99.py

```diff
@@ -1,12 +1,13 @@
 def solution(citations):
     a = []
     for h in range(max(citations)):
```

Comment:
h는 max(citations) 범위 내 일수도 있지만 len(citations)로도 할 수 있을 것 같습니다 ! 변경하면 반복의 횟수가 더 적어질 수 있을 것 같네요 :) 전체적으로 풀이 잘 하신 것 같습니다 
```suggestion
    for h in range(len(citations)):
```

Reply:
처음에는 ```len(citations)```를 사용했으나 오류가 떠서 바꿨습니다!
풀이의 조건을 보면 ```max()```를 사용해야 되더라구요 ㅠㅠ